Source/WebCore/css/parser/CSSParserContext.h

```diff
@@ -73,6 +73,7 @@ struct CSSParserContext {
     bool gradientPremultipliedAlphaInterpolationEnabled { false };
     bool gradientInterpolationColorSpacesEnabled { false };
     bool subgridEnabled { false };
+    bool shouldIgnoreImportRules { false };
```

Comment:
This should be included in operator==.

While the class is not super well organised it would still be good to put the bit further up. This section is for enable bits coming from settings.